extension/issues-prevnext.js

```diff
@@ -0,0 +1,29 @@
+/* globals utils */
+
+'use strict';
+
+const PREV_KEYCODE = 37;
+const NEXT_KEYCODE = 39;
+
+window.enableIssuesPrevNext = (() => {
+	const handler = ({keyCode, target}) => {
+		// just go to prev/next issue based on current issue number
+		if ((keyCode === PREV_KEYCODE || keyCode === NEXT_KEYCODE) && target.nodeName !== 'INPUT' && target.nodeName !== 'TEXTAREA') {
```

Comment:
This check _may_ be too generic. For users that prefer to navigate with keyboard shortcuts instead of the mouse, those keys are used to scroll left/right.